front/src/gc-components/GCList.tsx

```diff
@@ -0,0 +1,16 @@
+import { DefaultMantineColor, List, MantineNumberSize } from "@mantine/core";
+import { ReactNode } from "react";
+
+interface GCListProps {
+    children: ReactNode;
+    size?: MantineNumberSize;
+    color?: DefaultMantineColor;
+}
+
+export const GCList = ({ size, color, children }: GCListProps) => {
+    return (
+        <List size={size} color={color}>
```

Comment:
Here we also should use props in usuing original Mantine component
Plz check how we doind this on example of GCLink